awpy/parser/parse_demo.go

```diff
@@ -556,6 +559,19 @@ func convertRoundEndReason(r events.RoundEndReason) string {
 	}
 }
 
+func convertRoundMVPReason(r events.RoundMVPReason) string {
```

Comment:
This is probably one of the few functions that we can actually easily test on the go side. I feel like we should just do it.